packages/mui-joy/src/ToggleButtonGroup/ToggleButtonGroupContext.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+import * as React from 'react';
+
+interface IToggleButtonGroupContext {
+  onClick: (
```

Comment:
Let's make it optional so that the context value is empty by default:
```suggestion
  onClick?: (
```